website/content/docs/enterprise/replication.mdx

```diff
@@ -58,6 +60,28 @@ The following table shows a capability comparison between Disaster Recovery and
 | Mirrors the tokens and leases for applications and users interacting with the primary cluster                        | Yes               | No. Secondaries keep track of their own tokens and leases. When the secondary is promoted, applications must reauthenticate and obtain new leases from the newly-promoted primary. |
 | Allows the secondary cluster to handle client requests                                                               | No                | Yes                                            |
 
+Everything written to storage is classified into one of three categories:
+* `replicated` (or "shared"): all downstream clusters receive it
+* `local`: only downstream DR clusters receive it
+* `ignored`: not replicated to downstream clusters at all
+
+When mounting a secret engine or auth method, you can choose whether to make it a
+local mount or a shared mount.
+
+Shared mounts (the default) usually replicate all their data to PR secondaries, but
+they can choose to designate specific storage paths as local.  For example, PKI mounts
+store certificates locally. If you query the roles on a shared PKI mount, you'll see
+the same result for that mount when you send the query to either the PR primary or
+secondary, but if you list stored certs, you'll see different values.
+
+Local mounts replicate their data only to DR secondaries, and aren't visible at all to
+PR secondaries.
```

Comment:
Maybe it would be better to s/PR/Performance/ throughout?

It feels messy to me to use terms "DR" and "PR", since they use inconsistent conventions on how to abbreviate - the R in DR is "Recovery" but the R in PR is "Replication". If they were being abbreviated consistently, the counterpart to PR would be DRR ("Disaster Recovery Replication").

Reply:
I don't agree in principle with using the verbose terms, it feels too long-winded to me, but I'm applying your suggestion anyway for consistency with the rest of the documentation.

As for DR/PR being inconsistent names, fair point, but that ship has sailed, the terminology is pretty ingrained, at least internally.